modules/remove-central-attached-pv-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_content-type: CONCEPT
+[id="remove-central-attached-pv-overview_{context}"]
+= Remove Central-attached PV
+
+[role="_abstract"]
+Kubernetes and {ocp} do not delete persistent volumes (PV) automatically. When you upgrade {product-title-short} from earlier versions, the Central PV called `stackrox-db` remains mounted. However, in {product-title-short} 4.1, Central does not need the previously attached PV anymore.
+
+The PV has data and persistent files used by earlier {product-title-short} versions. You can use the PV to roll back to an earlier version before {product-title-short} 4.1. Or, if you have a large RocksDB Central backup bundle, you can use the PV to restore that data.
```

Comment:
```suggestion
The PV has data and persistent files used by earlier {product-title-short} versions. You can use the PV to roll back to an earlier version before {product-title-short} 4.1. Or, if you have a large `RocksDB Central` backup bundle, you can use the PV to restore that data.
```
See https://redhat-documentation.github.io/asciidoc-markup-conventions/ (suggest global change)

Reply:
[RocksDB](https://rocksdb.org/) is a product and therefore I want to keep the name as it is without the backticks. However, I've updated the sentence for more clarity:
```
... a large RocksDB backup bundle for Central, ...
```